test/script/run_tests.sh

```diff
@@ -38,7 +38,7 @@ setup_env () {
 }
 
 run_test_suite () {
-  standard
+  eslint .
```

Comment:
How come this doesn't need the `--ruledir` argument like we do in `package.json`?

Reply:
Yes we need it, forgot to update it here. May be i can do `npm run lint` instead?

Reply:
Yeah good idea, then you don't need to repeat the same line 👍